common/module/src/report/ACReporterHTML.tsx

```diff
@@ -18,7 +18,7 @@ import { IConfigInternal } from "../config/IConfig";
 import { Guideline } from "../engine/IGuideline";
 import { CompressedReport, IBaselineReport } from "../engine/IReport";
 import { GenSummReturn, IReporter, IReporterStored, ReporterManager } from "./ReporterManager";
-import { genReport } from "./genReport";
+import { genReport } from "../../../../accessibility-checker-extension/src/ts/util/htmlReport/genReport";
```

Comment:
```suggestion
import { genReport } from "./genReport";
```

This import shouldn't change. The report build process creates the genReport file in the right location. More generally, we shouldn't references files outside of the current src tree.